docs/src/environments/default-environment.md

```diff
@@ -1,284 +1,183 @@
 ---
 title: Rename the default environment
-description: |
-    See how to change the name of your default/production environment after creating a project. 
+description: See how to change the name of your default/production environment after creating a project.
+multipleTabs: true
 aliases:
   - /guides/general/default-branch.html
 ---
 
-You can set the name of your default/production branch when creating a project.
+You can set the name of your default/production environment when creating a project.
 To change it after project creation, follow the steps below.
+For consistency, the steps are all given using the [CLI](../administration/cli/_index.md).
 
-You can complete some of these steps through the Console,
-but since all can be completed with the CLI those commands alone are listed.
-Be sure to [install the CLI](../administration/cli/_index.md) if you haven't already done so.
-It's assumed you are changing the default branch of your project on Platform.sh from `master` to `main`.
-If using another name for the default branch, update the commands accordingly.
+The examples below are based off of changing the default environment from `old` to `main`.
+Replace these names with what suits your situation.
 
-{{< note >}}
+If your site is already live,
+remember that deactivating an environment is a destructive operation that can result in data loss.
+To minimize potential issues, take the following steps:
 
-If you have a domain set for your default environment, remove it before changing the default branch.
-Otherwise you get an error that `default domain must be a valid project domain`.
-
-{{< /note >}}
-
-## Projects without external integrations
-
-The steps below assume:
-
-- You already have a project on Platform.sh containing your code.
-- No external [source integrations](/integrations/source/_index.md) have been set up for the project.
-  That is, the Platform.sh project contains your primary remote repository for the site. 
-
-### 1. Create the Main environment
-
-To start, create a new environment off `master` called `main`:
-
-```bash
-$ platform environment:branch main master --title=Main -p <PROJECT_ID> --force
-```
-
-{{< note >}}
+- Switch the default environment during non-peak hours.
+- Keep your data by taking a [backup of the `old` environment](../administration/backup-and-restore.md)
+- Reduce your DNS time-to-live (TTL) to a minimum.
```

Comment:
:question: Why do we recommend lowering the TTL DNS settings? 
I assume that the `edge_hostname` needs to be updated on the DNS level (didn't check the whole process, but can do if needed), if so, that step is missing from the page.

Reply:
That's a good point. I guess it wasn't there in the previous one so I skipped it. Probably needs to be for live environments though. Added a step that I hope covers it.

Reply:
Wonderful, thank you! It should be clear enough now.